src/Makefile.am

```diff
@@ -483,8 +483,34 @@ crypto_libbitcoin_crypto_base_a_SOURCES = \
   crypto/sha3.h \
   crypto/sha512.cpp \
   crypto/sha512.h \
+  crypto/scrypt.cpp \
+  crypto/scrypt.h \
```

Comment:
How come crypto/scrypt_sse2.cpp was removed?  I see the scrypt_N_1_1_256_sp_sse2() function is still called in scrypt.cpp...